sklearn/multiclass.py

```diff
@@ -573,8 +573,10 @@ class OneVsOneClassifier(MetaEstimatorMixin, ClassifierMixin, BaseEstimator):
     Parameters
     ----------
     estimator : estimator object
-        An estimator object implementing :term:`fit` and one of
-        :term:`decision_function` or :term:`predict_proba`.
+        An estimator object implementing :term:`fit`. The check for
+        decision_function or predict_proba is left out because the
+        estimator can be from the regressors which do not have either
+        of these methods.
```

Comment:
Actually the original description was a bit misleading since indeed regressors can be used, but to accomodate them we should just _also_ check for `fit, predict` methods rather than _only_ checking for `fit`. Could you update your PR with the following:

1. Check that `estimator` has methods `fit` and one of `predict, predict_proba, decision_function`
2. Update the documentation of the `estimator` parameter to communicate this

Reply:
Actually, we can only check for `fit`. For classifier we will check for `decision_function` and fall-back on `predict_proba` if not available. For regressor then it will only use `predict`. So we cannot make a relevant check with the parameters.

I updated the documentation accordingdly.